tasks/main.yml

```diff
@@ -6,6 +6,7 @@
     - libgtk2.0-dev
     - libjasper-dev
     - libjpeg-dev
+    - liblapacke-dev
```

Comment:
what is `liblapacke`? is it just a weird name for `liblapack`? we should just be careful with these libraries because there are a few packages that compile against them, and changing them might have performance and accuracy implications